encoding/encoding_test.go

```diff
@@ -904,3 +904,33 @@ func generateFixedLenByteArrayValues(n int, r *rand.Rand, size int) []byte {
 	io.ReadFull(r, values)
 	return values
 }
+
+func maxLenInt8(data []int8) int {
+	max := 0
+	for _, v := range data {
+		if n := bits.Len8(uint8(v)); n > max {
+			max = n
+		}
+	}
+	return max
+}
+
+func maxLenInt32(data []int32) int {
```

Comment:
I'm not super clear on what this returns, given the name. If it's just the largest item I'd expect "maxInt32". If it's the length you'd call "len", what am I missing?

Reply:
"len" here is the length of the values in bits, as returned by `bits.Len*` functions in the standard `math/bits` package, I wanted to express the "maximum of all bits.Len in the list of int32".

Would you have a suggestion for a different name?